app/main/views/feedback.py

```diff
@@ -165,7 +165,7 @@ def thanks():
 
 
 def in_business_hours():
-    now = datetime.utcnow().replace(tzinfo=pytz.utc)
+    now = datetime.now(timezone.utc).replace(tzinfo=pytz.utc)
```

Comment:
Presumably it's TZ aware now we're not using utcnow
```suggestion
    now = datetime.now(timezone.utc)
```

Reply:
change applied